__tests__/app.test.js

```diff
@@ -0,0 +1,44 @@
+const request = require("supertest")
+const app = require("../app")
+const db = require("../db/connection")
+const seed = require("../db/seeds/seed")
+const data = require("../db/data/test-data")
+
+beforeAll(() => seed(data))
+
+afterAll(() => db.end())
+
+describe("GET/topics", () => {
+    test("200: Should respond with an array of objects containing topic data, with each object to contain a key of 'slug' and 'description'", () => {
+        return request(app)
+        .get("/api/topics")
+        .expect(200)
+        .then(({ body }) => {
+            const { topicsArray } = body;
+            topicsArray.forEach((topic) => {
+                expect(typeof topic.slug).toBe("string")
+                expect(typeof topic.description).toBe("string")
+            })
+            expect(topicsArray[0].slug).toBe('mitch')
+            expect(topicsArray[1].slug).toBe('cats')
+            expect(topicsArray[2].slug).toBe('paper')
+            expect(topicsArray[0].description).toBe('The man, the Mitch, the legend')
+            expect(topicsArray[1].description).toBe('Not dogs')
+            expect(topicsArray[2].description).toBe('what books are made of')
```

Comment:
specific assertions like this are not needed for a GET all test

We only really want to go into specifics when we're getting a specific item back.

One check you might want to add in, though, is a check to see if the arr of topics isn't empty/has the right number of items in it. Because if it was empty the forEach would give us a false positive